.travis.yml

```diff
@@ -42,11 +41,19 @@ before_install:
     ${run_opts} ${distribution}-${version}:ansible "${init}"
 
 install:
-  - 'pip install -r requirements-devel.txt'
+  - 'sudo apt-get update -qq'
+  - 'sudo apt-get install -y python-pip'
+  - 'sudo pip install -r requirements-devel.txt'
+  - 'sudo pip install ansible'
```

Comment:
I'm pretty sure we don't need this, package `ansible` is only needed inside the container.